aws-mssql.yml

```diff
@@ -11,8 +11,31 @@ support_url: https://aws.amazon.com/sql/
 tags: [aws, mssql, beta]
 plan_updateable: true
 provision:
-  plan_inputs: [] # Expected: mssql_version, storage_gb
+  plan_inputs: []
   user_inputs:
+  - field_name: engine
+    required: true
```

Comment:
question: does it make sense to combine the default value with the required parameter?

Reply:
Very good point.
Two opposing points of view.

- Adding both is a good documentation for future us.
If someone ever decides the remove the default 
the required property will still be there to remember
them that the field is required.
- Having a default value for a required field makes it behaves
as if it wasn't required at all. This might make reasoning about it
more difficult both in tests and maybe in production.

I believe still it would be possible for a plan to enforce instances
to specify the field explicitly (without the default).
To do that, the plan should explicitly specify `engine: null`.

Personally, I like the idea of specifying `required: true` even if there is a
default value. Although, I believe I introduced this inconsistency by accident.

Reply:
Thanks, Andrea.
In the end, it seemed reasonable to not have a default value for `engine` field at all.